xbmc/settings/DisplaySettings.cpp

```diff
@@ -792,6 +792,26 @@ void CDisplaySettings::SettingOptionsMonitorsFiller(SettingConstPtr setting, std
     }
     list.push_back(std::make_pair(monitors[i], monitors[i]));
   }
+#elif defined(HAVE_WAYLAND)
```

Comment:
You could set a monitor that you unplug then. Or you just set `monitor` to some made-up value in `guisettings.xml`.